### PR TITLE
[INDY-1135] Added GET_REVOC_REG_DEF transaction

### PR DESCRIPTION
**Pull request by anikitinDSR**
Signed-off-by: Andrew Nikitin <andrew.nikitin@dsr-corporation.com>

### CODE REVIEW

**Comment by hyperledger-github** [comment]
Could one of the committers please verify this patch?

File: indy_node/server/domain_req_handler.py [hunk 1]
```diff
@@ -369,6 +371,20 @@ def handleGetClaimDefReq(self, request: Request):
         result[SIGNATURE_TYPE] = signatureType
         return result
 
+    def handleGetRevocRegDefReq(self, request: Request):
+        state_path = request.operation.get(ID, None)
+        assert state_path
+        try:
+            keys, last_seq_no, last_update_time, proof = self.lookup(state_path, isCommitted=False)
```
Comment (ashcherbakov) [inline indy_node/server/domain_req_handler.py]: `isCommitted` must be True. Please add test for this.

File: indy_node/test/anon_creds/test_get_revoc_reg_def_send.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                     domain.MARKER_REVOC_DEF,
+                      revoc_reg['operation'][CRED_DEF_ID],
+                      revoc_reg['operation'][TYPE],
+                      revoc_reg['operation'][TAG]]),
+        TYPE: GET_REVOC_REG_DEF,
+    }
+    get_revoc_reg_def_req = sdk_sign_request_from_dict(looper,
+                                                       sdk_wallet_steward,
+                                                       get_revoc_reg_def_req)
+    sdk_send_and_check([json.dumps(get_revoc_reg_def_req)],
```
Comment (ashcherbakov) [inline indy_node/test/anon_creds/test_get_revoc_reg_def_send.py]: Please check the returned content (that we really got a correct and expected RevocRegDef)

File: indy_node/test/anon_creds/test_get_revoc_reg_def_send.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    }
+    get_revoc_reg_def_req = sdk_sign_request_from_dict(looper,
+                                                       sdk_wallet_steward,
+                                                       get_revoc_reg_def_req)
+    replies = sdk_send_and_check([json.dumps(get_revoc_reg_def_req)],
+                                 looper,
+                                 txnPoolNodeSet,
+                                 sdk_pool_handle)
+    req, reply = replies[0]
+    assert revoc_reg_def_id == reply['result']['data'][ID]
```
Comment (ashcherbakov) [inline indy_node/test/anon_creds/test_get_revoc_reg_def_send.py]: Please check all data, not just ID

**Pull request merged**